### DISCUSSION TITLE
Add prettier config file as each system - Per pr #1156

### DISCUSSION THREAD

**Discussion by mtfoley** [open-sauced/open-sauced#1164] [Ideas]
_Add prettier config file as each system has it's own config. Due to which a large portion of a file gets changed._

The goal would be so that the contributor's system preferences of prettier utility don't make formatting changes that offend our eslint rules.

**Comment by 0-vortex** [comment]
We cannot stop users from not using our configuration, we have a lint and format check in actions, it’s just a matter of enforcing all the rules that we want - from eslintrc first as prettier is a subset of what we have right now
